change/@fluentui-web-components-2f7edf54-0877-419b-8a74-47bf7c3681f8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31157&commitId=031694a6b3a1907977c95aa6b4cdb93a26024242&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.With Long Text - RTL.default.chromium.png|1686| Changed|
|Button.With Long Text.pressed.chromium.png|1460| Changed|
|Button.With Long Text.default.chromium.png|1703| Changed|
|Button.With Long Text.hover.chromium.png|1624| Changed|
|Button.With Long Text - RTL.hover.chromium.png|1607| Changed|
|Button.With Long Text - RTL.pressed.chromium.png|1445| Changed|

</details>


  <details><summary><b>Slider</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Slider Steps Vertical.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Text</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Block.normal.chromium.png|995| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>